spec/spec.yaml

```
@@ -77,6 +77,11 @@ tags:
       A release provides _components_, like for example Kubernetes. For each
       release the contained components are listed. Changes in components are
       detailed in the _changelog_ of a release.
+  - name: cluster labels
+    description: |
+      Cluster labels allow users to group and tag clusters with custom key-value-pairs.
+
+      Note: Cluster labelling is currently (as of February 2020) only available on cluster releases 10 or higher on <span class="badge aws">AWS</span>.
```

```suggestion
      Note: Cluster labelling is currently only available on release version 10 or higher on <span class="badge aws">AWS</span>.
```

We can update this with corrected info when the time comes, I don't think we need to have the date in here necessarily